boolean/boolean.py

```diff
@@ -971,7 +971,7 @@ def __init__(self, *args):
 
     def __init__(self, arg1):
         super(NOT, self).__init__(arg1)
-        self.isliteral = self.args[0].isliteral
+        self.isliteral = isinstance(self.args[0], Symbol)
```

Comment:
Same as above.. if args[0] fails, it's probably due to another bug. We're in the context of a Not here.